src/telliot_feeds/reporters/reporter_autopay_utils.py

```diff
@@ -122,14 +123,25 @@ async def get_current_feeds(self, require_success: bool = False) -> Any:
                     )
                 )
         multi_call = Multicall(calls=calls, _w3=self.w3, require_success=require_success)
-        data = await multi_call.coroutine()
+        try:
+            data: Dict[str, Any] = await multi_call.coroutine()
```

Comment:
you don't need to define the types for `data`

Reply:
nvm, if you want to make types clearer, idk why we shouldn't do that

Reply:
yeah `mypy` wanted it, but yes i think it also increases code readbility when interacting with outside libraries